src/ui/targetingTree.ts

```diff
@@ -156,4 +165,16 @@ const insertScheduleTree = (rootTree: Tree, rollout?: Rollout) => {
 
         rootTree.insert('gradual rollout', scheduleTree)
     }
+}
+
+const coloredTitle = (title: string) => {
+    return chalk.hex('#93E3FE')(title)
```

Comment:
it seems like it'd be worthwhile defining these hex codes as constants, maybe in the UI folder? then we can easily reuse these colours